pypit/core/artraceslits.py

```diff
@@ -990,10 +981,12 @@ def edgearr_close_slits(binarr, edgearrcp, edgearr, ednum, settings):
     edgearrcp : ndarray
       Which was modified in place
     """
+
     vals = np.sort(np.unique(edgearrcp[np.where(edgearrcp != 0)]))
-#        print('calling close_edges')
-#        exit()
-    hasedge = arcytrace.close_edges(edgearrcp, vals, int(settings['trace']['slits']['maxgap']))
+#    hasedge = arcytrace.close_edges(edgearrcp, vals, int(settings['trace']['slits']['maxgap']))
+    # TODO: This pure python function was never tested!
```

Comment:
There are three instances of this TODO item that I think would be best to clarify... I think we should clarify that these functions were converted from cython, and haven't been tested since this conversion.

Reply:
Done!